core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -634,6 +634,13 @@
  * <tr>
  * <td>N/A</td>
  * <td>N/A</td>
+ * <td>{@link #METRICS_MANAGER_COMPACTION_SVC_ERRORS}</td>
+ * <td>Gauge</td>
+ * <td></td>
```

Comment:
Would be nice to fill in the description.

Reply:
According to the table header, that column is `Notes`. And the table description is `Summary of Metric Changes`. I created the table in the javadoc to help people map from the old Hadoop Metrics2 to Micrometer as that change was done in 2.1. Most of the values in that column are empty.  Maybe we should modify the javadoc table in 3.1 and populate it with descriptions in some manner that can be used in the user guide.

Reply:
It's sort of referenced in the user guide. There is a link to this table: https://accumulo.apache.org/docs/2.x/administration/monitoring-metrics#metric-names 

I agree that modifying the table and adding the descriptions would be a good improvement.

Reply:
ok, I thought that column was a description.  Makes sense to leave it blank.

Reply:
Created #4815